test/fakeintake/client/client.go

```diff
@@ -354,7 +355,7 @@ func (c *Client) FilterMetrics(name string, options ...MatchOpt[*aggregator.Metr
 	return filteredMetrics, nil
 }
 
-// WithTags filters by `tags`
+// WithTags filters by `tags`.
```

Comment:
Other comments don’t end with a trailing dot.

```suggestion
// WithTags filters by `tags`
```